src/main.browser.ts

```diff
@@ -0,0 +1,37 @@
+import { initializeBKTClientInternal } from './BKTClient'
+import { BKTConfig } from './BKTConfig'
+import { BKTUser } from './BKTUser'
+import { Component, DefaultComponent } from './internal/di/Component'
+import { DataModule } from './internal/di/DataModule'
+import { InteractorModule } from './internal/di/InteractorModule'
+import { BrowserPlatformModule } from './internal/di/PlatformModule.browser'
+import { User } from './internal/model/User'
+
+export type { BKTConfig } from './BKTConfig'
+export { defineBKTConfig } from './BKTConfig'
+export type { BKTUser } from './BKTUser'
+export { defineBKTUser } from './BKTUser'
+export type { BKTClient } from './BKTClient'
+export { getBKTClient, destroyBKTClient } from './BKTClient'
+export type {
+  BKTStorage,
+  BrowserLocalStorage,
+  InMemoryStorage,
+} from './BKTStorage'
+
+const createBrowserComponent = (config: BKTConfig, user: User): Component => {
+  return new DefaultComponent(
+    new BrowserPlatformModule(),
+    new DataModule(user, config),
+    new InteractorModule(),
+  )
+}
+
+export const initializeBKTClient = async (
```

Comment:
I moved `initializeBKTClient()` method to these entry points, so that we can change its implementation.